rules/rules.js

```diff
@@ -108,6 +108,55 @@ const removeRule = function (uid) {
     }
 };
 
+/**
+ * Runs the rule with the given UID. Throws errors when the rule doesn't exist
+ * or is unable to run (e.g. it's disabled);
+ * 
+ * @memberOf rules
+ * @param {String} uid the UID of the rule to run
+ * @param {Map<Object>} args optional dict of data to pass to the called rule
+ * @param {Boolean} cond when true, the called rull will only run if it's conditions are met
```

Comment:
Typo: should be rule and not rull

Reply:
`args` and `cond` are optional and have defaults.
You can adjust the JSDoc to show that:
```js
/** 
 * …
 * @param {Map<Object>} [args={}] optional dict …
 * @param {Boolean} [cond=true] when true, …
 */
```

Reply:
I'll fix it.

Reply:
Done

Reply:
Did you fix that? I can‘t see changes.